server/bin/endpoints/utils.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:convert' as convert;
+import 'dart:io';
+import 'dart:math';
+
+import 'package:http/http.dart' as http;
+
+String getFirebasePath() {
+  const firebasePath = String.fromEnvironment('FIREBASE_PATH');
+  if (firebasePath.isEmpty) {
+    stdout.writeln('Could not parse compile time variable FIREBASE_PATH.');
+    exitCode = 64;
+    return '';
+  }
+  return firebasePath;
+}
+
+Future<String> listDocuments(String firebase, String path) async {
+  final url = Uri.https('firestore.googleapis.com',
+      'v1/projects/$firebase/databases/(default)/documents/$path');
+  return (await http.get(url)).body;
+}
+
+Future<List<String>> listIDocumentIds(String firebase, String path) async {
```

Comment:
there is a `I` in the middle of `list*I*DocumentIds`, is it a typo?

Reply:
probably

Reply:
fixed